frmts/stacit/stacitdataset.cpp

```diff
@@ -594,34 +607,37 @@ bool STACITDataset::SetupDataset(
             poVRTBand->SetColorInterpretation(eInterp);
 
         // Set band properties
-        if (asset.eoBands.IsValid() &&
-            asset.eoBands.Size() == poItemDS->GetRasterCount())
+        if (asset.bands.IsValid() &&
+            asset.bands.Size() == poItemDS->GetRasterCount())
         {
-            const auto &eoBand = asset.eoBands[i];
-            const auto osBandName = eoBand["name"].ToString();
+            const auto &band = asset.bands[i];
+            const auto osBandName = band["name"].ToString();
             if (!osBandName.empty())
                 poVRTBand->SetDescription(osBandName.c_str());
 
             if (eInterp != GCI_Undefined)
             {
-                const auto osCommonName = eoBand["common_name"].ToString();
+                std::string osCommonName;
+                if (band.Has("eo:common_name"))
+                    osCommonName = band["eo:common_name"].ToString();
+                else
+                    osCommonName = band["common_name"].ToString();
                 if (osCommonName == "red")
                     poVRTBand->SetColorInterpretation(GCI_RedBand);
                 else if (osCommonName == "green")
                     poVRTBand->SetColorInterpretation(GCI_GreenBand);
                 else if (osCommonName == "blue")
                     poVRTBand->SetColorInterpretation(GCI_BlueBand);
-                else if (osCommonName == "alpha")
-                    poVRTBand->SetColorInterpretation(GCI_AlphaBand);
```

Comment:
There is no common name called 'alpha" in STAC.

Reply:
Really ? but please don't remove that there, otherwise this is going to conflict with PR https://github.com/OSGeo/gdal/pull/10721 that touches that code too

Reply:
Yes, no alpha. The full list of allowed values is here: https://github.com/stac-extensions/eo?tab=readme-ov-file#common-band-names

Reply:
The list was also extended a bit recently and the property in STAC 1.1 is a named eo:common_name and can be placed in asset or bands.

Reply:
> The list was also extended a bit recently

yes, #10721 takes inspiration from it, but feedback of the GDAL community was that some of the details like "rededge071", "rededge075", etc. were much too specific, so we didn't go to that detail for the GCI_ColorInterpretation enumeration.

I'm wondering if #10721 shouldn't be merge first (should be soon. I'm just letting a couple extra days for potential new comments on it), so that you can work on a stable base

Reply:
Yeah, go ahead. This will still take me a little longer anyway.